mon-pix/translations/fr.json

```
@@ -1222,7 +1222,8 @@
         "starsAcquired": "{acquired, plural, =0 {aucune étoile acquise} other {# étoiles acquises}} sur {total}"
       },
       "net-promoter-score": {
-        "title": "Votre avis compte ! Prenez quelques minutes pour nous dire ce que vous avez pensé de ce test et nous aider à l'améliorer.",
+        "title": "Votre avis compte !",
+        "text": "Prenez quelques minutes pour nous dire ce que vous avez pensé de ce test et nous aider à l'améliorer.",
         "link": {
           "label": "Je donne mon avis",
           "href": "https://pix.fr"
```

```suggestion
             "net-promoter-score": {
        "title": "Votre avis compte !",
        "link": {
          "label": "Je donne mon avis",
          "href": "https://pix.fr"
        },
        "text": "Prenez quelques minutes pour nous dire ce que vous avez pensé de ce test et nous aider à l'améliorer."
      }
```

Suggestion: Inverser l'ordre des entrées dans le JSON pour conserver l'ordre alphabétique mis en place (excepté `title` qui reste en tête)